recipe/meta.yaml

```
@@ -27,13 +27,13 @@ requirements:
     - pip
   run:
     - python >=3.8
-    - dask-core ==2023.1.1
-    - distributed ==2023.1.1
+    - dask-core ==2023.3.2
+    - distributed ==2023.3.2.1
     - dask ==2023.1.1
```

Seems like the issue here is just a mismatched `dask` pinning?

```suggestion
    - dask ==2023.3.2
```

Not sure if this is something we would've expected the bot to pick up; FWIW if we merge in https://github.com/dask/dask/pull/10174 and https://github.com/dask/distributed/pull/7769 we shouldn't encounter the issues with Dask nightlies anymore and should be able to remove the `dask-core`/`distributed` pinnings altogether